examples/hotrod/services/frontend/server.go

```diff
@@ -54,20 +56,27 @@ func NewServer(options ConfigOptions, tracer opentracing.Tracer, logger log.Fact
 		logger:   logger,
 		bestETA:  newBestETA(tracer, logger, options),
 		assetFS:  assetFS,
+		basepath: options.Basepath,
 	}
 }
 
 // Run starts the frontend server
 func (s *Server) Run() error {
 	mux := s.createServeMux()
-	s.logger.Bg().Info("Starting", zap.String("address", "http://"+s.hostPort))
+	s.logger.Bg().Info("Starting", zap.String("address", "http://"+s.hostPort+s.basepath))
```

Comment:
use `path.Join`, i.e. `"http://"+path.Join(s.hostPort, s.basepath)`. Join will prevent duplicate `/`, so a `basepath` of `foo` and `/foo` are both valid. This will make the implementation more robust.